foundations/javascript_basics/understanding_errors.md

```diff
@@ -114,9 +114,9 @@ Here, we get a `TypeError` with a message stating that `str1.push is not a funct
 
 At this point, you might be wondering how we can resolve these errors.
 
-1.  Read the error *carefully* and try to understand it on your own.
+1.  Read the error _carefully_ and try to understand it on your own.
 2.  Next, Google the error! Chances are, you can find a fix or explanation on StackOverflow or in the documentation. If nothing else, you will receive more clarity as to why you are receiving this error.
-3.  Use the debugger! As previously mentioned, the debugger is great for more involved troubleshooting, and is a critical tool for a developer. You can set breakpoints, view the value of any given variable at any point in your application’s execution, step through code line by line, and more! It is an extremely valuable tool and every programmer should know how to use it. [This tutorial dives into the Chrome Debugger](https://developer.chrome.com/docs/devtools/javascript/).
+3.  Use the debugger! As previously mentioned, the debugger is great for more involved troubleshooting, and is a critical tool for a developer. You can set breakpoints, view the value of any given variable at any point in your application’s execution, step through code line by line, and more! It is an extremely valuable tool and every programmer should know how to use it. [This tutorial dives into the Chrome Debugger](https://developer.chrome.com/docs/devtools/javascript/). Warning: In point 4 of step 3 of the tutorial, it is mentioned devtools will pause at the line where the function is declared but we are aware sometimes it may pause on the second line of the declaration. Dont worry this is a [known issue](https://github.com/GoogleChrome/developer.chrome.com/issues/2947).
```

Comment:
Apologies for the unclear instructions. The latest comment in the issue you're creating this PR from states the issue with the article itself. Can we therefore modify this warning just to mention that it should pause on the second line so not to worry.

Reply:
All good, thank you for your feedback. I have resubmitted the PR.